pkg/provider/azure_privatelinkservice.go

```diff
@@ -451,11 +451,8 @@ func reconcilePLSVisibility(
 	service *v1.Service,
 ) (bool, error) {
 	changed := false
-	visibilitySubs, anyoneCanView := getPLSVisibility(service)
+	visibilitySubs, _ := getPLSVisibility(service)
```

Comment:
Since it is the only place to call `getPLSVisibility(), why not just return `visibilityList`?

Reply:
I want to keep it in case it's needed in the future.